runtests.py

```diff
@@ -21,4 +21,5 @@
 django.setup()
 
 if __name__ == "__main__":
-    call_command('test', 'widget_tweaks', verbosity=2)
+    param = "" if not len(sys.argv) >= 2  else "." +str(sys.argv[1])
```

Comment:
Could we have a comment here about the functionality?

The two cases should most probably be handled with an if for the sake of readability

    # Allow passing test name as argument e.g.
    # python runtests.py ...
    if len(sys.argv) >= 2:
        call_command(...)
    else:
        call_command('test', 'widget_tweaks', verbosity=2)

Reply:
I'll add the comments to upstream.

Reply:
Done in https://github.com/jazzband/django-widget-tweaks/commit/363e26b0f8de4d3376ce07357974e90ea67c123e